.github/workflows/build-and-push-image-build.yml

```diff
@@ -3,14 +3,41 @@ name: Build and push access-nri/build images
 on:
   workflow_dispatch:
     inputs:
+      spack-packages-build-required:
```

Comment:
When/where is this dispatched?

Reply:
this will be an option when running a workflow from github - it'll just be a timesaver in the case where a model is updated, and we want to use an existing tagged base-spack image so we can reuse rather than rebuild every time. 
Was that what you meant?

Reply:
Up to now it was only dispatched manually, is that right?